### PR TITLE
Notify user of thread deletion through channel message

### PR DESCRIPTION
**Pull request by OTCompa**
Ticket: MOD-47
This PR updates Moddingway to notify the user of their thread being automodded in the specified channel (#bot-channel)

### CODE REVIEW

File: src/settings.py [hunk 1]
```diff
@@ -16,12 +16,14 @@ class Settings(BaseModel):
     postgres_username: str = os.environ["POSTGRES_USER"]
     postgres_password: str = os.environ["POSTGRES_PASSWORD"]
     automod_inactivity: dict[int, int]  # key: channel id, value: inactive limit (days)
+    notify_channel_id: int
```
Comment (brtran4) [inline src/settings.py:19]: very small nit, let's group this with `logging_channel_id`

File: src/settings.py [hunk 2]
```diff
@@ -51,6 +53,9 @@ def local() -> Settings:
         postgres_host=os.environ.get("POSTGRES_HOST", "localhost"),
         postgres_port=os.environ.get("POSTGRES_PORT", "5432"),
         automod_inactivity=automod_inactivity,
+        notify_channel_id=os.environ.get(
+            "NOTIFY_CHANNEL_ID", os.environ["MOD_LOGGING_CHANNEL_ID"]
```
Comment (brtran4) [inline src/settings.py:57]: lol another var to add to the readme

File: src/workers/helper.py [hunk 3]
```diff
@@ -66,34 +67,33 @@ async def automod_thread(
...
-        #             f'Your thread "{thread.name}" in <#{channel_id}> has been automatically deleted as {duration} days have passed without any activity or the starter message has been deleted.',
-        #         )
-        #     except discord.Forbidden:
-        #         raise UnableToDM("User has DMs disabled")
-        # else:
-        #     raise UnableToDM("User is not in the server")
+        if thread.owner is not None:
+            try:
+                await notifying_channel.send(
+                    f'{thread.owner.mention}, your thread "{thread.name}" in <#{channel_id}> has been automatically deleted as {duration} days have passed without any activity or the starter message has been deleted.'
```
Comment (brtran4) [inline src/workers/helper.py:78]: @clairefleuret text check

File: src/workers/helper.py [hunk 4]
```diff
@@ -66,34 +67,33 @@ async def automod_thread(
...
         channel = self.get_channel(settings.logging_channel_id)
         async with create_interaction_embed_context(
             channel,
             user=thread.owner,
             timestamp=datetime.now(timezone.utc),
-            description=f"<@{thread.owner_id}>'s thread was deleted in <#{channel_id}> but a DM could not be sent: {e}",
+            description=f"<@{thread.owner_id}>'s thread was deleted in <#{channel_id}> but a notification could not be sent: {e}",
         ):
             pass
+        # not really an error we're worried about since it's just notifying
```
Comment (brtran4) [inline src/workers/helper.py:95]: nit: `NB`

**Review by brtran4 — CHANGES_REQUESTED** [review]

**Review by brtran4 — APPROVED** [review]

File: src/workers/helper.py (hunk 3 above)
Comment (clairefleuret) [reply src/workers/helper.py:78]: Should be “your original message” at the end and “your post” not thred

**Pull request merged**